diff -aur ViennaRNA-2.1.7/RNAforester/src/rnafuncs.h ViennaRNA-2.1.7.new/RNAforester/src/rnafuncs.h
--- ViennaRNA-2.1.7/RNAforester/src/rnafuncs.h	2014-04-12 21:23:20.000000000 +0200
+++ ViennaRNA-2.1.7.new/RNAforester/src/rnafuncs.h	2014-07-25 18:34:46.433574503 +0200
@@ -39,7 +39,7 @@
     map<int,string> synonyms;
     Uint xbasepos,ybasepos;
     bool xmlInput;
-  };
+  } AddXmlInfos;
 
   static bool isRNAString(const string &str);
   static bool isViennaString(const string &str, Ulong &basePairCount, Ulong &maxDepth);
